UnityProject/Assets/Scripts/Managers/LobbyManager/MusicManager.cs

```diff
@@ -111,7 +116,7 @@ public static bool isLobbyMusicPlaying()
 		{
 			if (Instance.currentLobbyAudioSource != null
 			    && Instance.currentLobbyAudioSource.isPlaying
-			    || !(SunVox.sv_end_of_song((int) Slot.Music) == 1))
+			    || !(SunVox.sv_end_of_song((int) Slot.Music) == 0))
```

Comment:
Why did this change from 1 to 0?

Reply:
Basically, because it was backwards.  sv_end_of_song returns 0 if the song is still playing, 1 if the song has ended, as in is not playing (I know, its weird, but it makes sense... kind of).  As it was, this statement would return true if either the current lobby audiosource was playing or SunVox was not playing, ie it would be true whether a song was playing or not, and could only be false if both SunVox was playing and a song wasn't playing, ie it could never be false.

Reply:
fair enough